Mail and ocr

Xalt8 opened the pull request:
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Reviewers:

Please add the GitHub usernames of the reviewers:

- @username1
- @username2
- @username3

## Screenshots (if appropriate):

## Additional context

Add any other context or screenshots about the pull request here.

Xalt8 commented:
The summary function is working. The OCR for attachments is yet to be incorporated.

isayahc commented on fetch_mail/Attachments/718ba392-d9e1-4e4b-80b3-b30809387e07_istockphoto-2149237610-1024x1024.jpg:
could we add image to the `examples` directory.

isayahc commented on fetch_mail/emails.json:
for the file names. such as

`
"attachments": [
            "D:\\singh\\Documents\\Mail_Automation_WF\\mail_automation_wf\\Fetch Mail\\Attachments\\718ba392-d9e1-4e4b-80b3-b30809387e07_istockphoto-2149237610-1024x1024.jpg",
            "D:\\singh\\Documents\\Mail_Automation_WF\\mail_automation_wf\\Fetch Mail\\Attachments\\718ba392-d9e1-4e4b-80b3-b30809387e07_Sample filled Claim Form.pdf"
        ]`
Would we be able to turn this absolute path into a relative path like 


`"\\Mail_Automation_WF\\mail_automation_wf\\Fetch Mail\\Attachments\\718ba392-d9e1-4e4b-80b3-b30809387e07_Sample filled Claim Form.pdf"`

isayahc reviewed (changes requested):
Nothing major just changing to make sure the paths works for other machines.

Xalt8 commented:
1. I've deleted the output_dir
2. The attachment location is not now relative referenced
3.  Added an image to examples
4. Changed the uuid identifier to use email entry id
